IBPSA/Resources/Scripts/Dymola/Airflow/Multizone/Validation/DoorOpenClosed.mos

```diff
@@ -0,0 +1,3 @@
+simulateModel("IBPSA.Airflow.Multizone.Validation.DoorOpenClosed", method="CVode", stopTime=1, tolerance=1e-06, resultFile="DoorOpenClosed");
+createPlot(id=1, position={15, 15, 799, 543}, y={"doo.port_a1.m_flow", "doo.port_b2.m_flow", "dooOpeClo.port_a1.m_flow", "dooOpeClo.port_b2.m_flow"}, range={0.0, 1.0, -1.0, 3.0}, grid=true, colors={{28,108,200}, {238,46,47}, {0,140,72}, {217,67,180}});
+createPlot(id=1, position={15, 15, 799, 543}, y={"lea.port_a.m_flow", "mNetClo_flow.y"}, range={0.0, 1.0, -0.005, 0.009999999999999998}, grid=true, subPlot=102, colors={{28,108,200}, {238,46,47}});
```

Comment:
Is this `subplot=102` intentional? Dymola 2020 does not like it:

<img width="809" alt="image" src="https://user-images.githubusercontent.com/5970879/105756699-3c140e80-5f4d-11eb-9739-a923f16ab6c3.png">

Reply:
@Mathadon : The `subPlot=102` is not intentional. I can fix it tomorrow. Thanks for catching it.

Reply:
Just to be sure: the other scripts have it too.